sdcm/cluster_aws.py

```diff
@@ -405,49 +405,12 @@ def _node_setup(self, node):
             time.sleep(5)
             node.remoter.run('nodetool status', verbose=True)
 
-    def wait_for_init(self, node_list=None, verbose=False):
```

Comment:
Can we set a big default timeout to avoid the stuck in special case?
Such as 120 mins?

@roydahan

Reply:
For now we handle it only for specific case, add node after decommission.
Let's see if we have other cases were the test is stuck before we starts to add more timeouts.